.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
It seems you might have missed the test.md file to your gitignore, which is a requirement

[    - A test.md file with the text "This file should be ignored by git".](https://github.com/microverseinc/curriculum-transversal-skills/blob/5cb3d2611298d05ddb5eaf2abf41524555639a76/documentation/hello_microverse_project.md?plain=1#L44)